samples/word/50-document/manage-custom-xml-part.yaml

```
@@ -0,0 +1,222 @@
+order: 7
+id: word-document-manage-custom-xml-part
+name: Manage a CustomXmlPart
+description: 'This sample shows how to add, query, edit, and delete a custom XML part in a document.'
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#add-custom-xml-part").click(() => tryCatch(addCustomXmlPart));
+        $("#query").click(() => tryCatch(query));
+        $("#insert-attribute").click(() => tryCatch(insertAttribute));
+        $("#insert-element").click(() => tryCatch(insertElement));
+        $("#delete-custom-xml-part").click(() => tryCatch(deleteCustomXmlPart));
+
+        async function addCustomXmlPart() {
+          // Add a custom XML part.
+          await Word.run(async (context) => {
+            // You must have the xmlns attribute to populate the CustomXmlPart.namespaceUri property.
+            const originalXml =
+              "<Reviewers><Reviewer>Juan</Reviewer><Reviewer>Hong</Reviewer><Reviewer>Sally</Reviewer></Reviewers>";
+            const customXmlPart = context.document.customXmlParts.add(originalXml);
+            customXmlPart.load("id");
+            const xmlBlob = customXmlPart.getXml();
+
+            await context.sync();
+
+            const readableXml = addLineBreaksToXML(xmlBlob.value);
+            console.log("Added custom XML part:");
+            console.log(readableXml);
+
+            // Store the XML part's ID in a setting so the ID is available to other functions.
+            const settings = context.document.settings;
+            settings.add("ContosoReviewXmlPartId", customXmlPart.id);
+
+            await context.sync();
+          });
+        }
+
+        async function query() {
+          // Query the specified custom XML part.
+          await Word.run(async (context) => {
+            const settings = context.document.settings;
+            const xmlPartIDSetting = settings.getItemOrNullObject("ContosoReviewXmlPartId").load("value");
+
+            await context.sync();
+
+            if (xmlPartIDSetting.value) {
+              const customXmlPart = context.document.customXmlParts.getItem(xmlPartIDSetting.value);
+              const toQueryFor = "/Reviewers/Reviewer";
+              const clientResult = customXmlPart.query(toQueryFor, {
+                contoso: "http://schemas.contoso.com/review/1.0"
+              });
+
+              await context.sync();
+
+              console.log(`Queried custom XML part for ${toQueryFor} and found ${clientResult.value.length} matches:`);
+              for (let i = 0; i < clientResult.value.length; i++) {
+                console.log(clientResult.value[i]);
+              }
+            } else {
+              console.warn("No custom XML part to query");
+            }
+          });
+        }
+
+        async function insertAttribute() {
+          // Insert an attribute to the specified custom XML part.
```

"to" doesn't seem like the right preposition to me, but I'm not as familiar with XML grammar.
```suggestion
          // Insert an attribute in the specified custom XML part.
```